ballerina/client_utils.bal

```diff
@@ -98,10 +98,51 @@ type OpenAICreateChatCompletionRequest record {
     boolean? 'stream = false;
     decimal? temperature = 1;
     decimal? top_p = 1;
+    ChatCompletionTool[] tools?;
+    ChatCompletionToolChoiceOption tool_choice?;
+};
+
+type ChatCompletionToolChoiceOption ChatCompletionNamedToolChoice;
```

Comment:
Shall we follow the rest of the file and move the definitions to before they are used?